tests/unit/test_schemas.py

```diff
@@ -8,13 +8,16 @@
 import string
```

Comment:
As noted above --- let's just delete this module. It's not testing any code we've written, so it's a bit pointless.